Library/Formula/glibc.rb

```diff
@@ -4,6 +4,8 @@ class Glibc < Formula
   url "http://ftpmirror.gnu.org/glibc/glibc-2.19.tar.bz2"
   sha256 "2e293f714187044633264cd9ce0183c70c3aa960a2f77812a6390a3822694d15"
 
+  option "enable-kernel", "Compile for compatibility with kernel not older than your current one" if OS.linux?
```

Comment:
`glibc` is a Linux-only package, so the `if OS.linux?` guards are not necessary in this case.